src/OmniSharp.Nuget/project.json

```diff
@@ -16,17 +16,6 @@
         "NuGet.Packaging": "3.5.0-beta2-1484",
         "NuGet.Packaging.Core": "3.5.0-beta2-1484",
         "Microsoft.Web.Xdt": "2.1.1"
```

Comment:
I'm leaving it so @david-driscoll can have a go at it. At the moment, it isn't actually built as part of OmniSharp.